test/test_dgmulti_2d.jl

```diff
@@ -16,26 +16,26 @@ isdir(outdir) && rm(outdir, recursive=true)
   @trixi_testset "elixir_euler_weakform.jl" begin
     @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_weakform.jl"),
       cells_per_dimension = (4,4),
-      l2 = [0.0013463150267220328, 0.0014235793662296975, 0.0014235793662300024, 0.00472191786388071],
-      linf = [0.0015248303482329195, 0.0020707330926952316, 0.002070733092696342, 0.004913455679613321],
+      l2 = [0.0013536930300254945, 0.0014315603442106193, 0.001431560344211359, 0.0047393341007602625],
+      linf = [0.001514260921466004, 0.0020623991944839215, 0.002062399194485476, 0.004897700392503701],
```

Comment:
So the `cfl` set in these elixirs was too loose in some sense? I'm just wondering since you didn't change the `cfl` number there.

Reply:
Yes, it was pretty loose - I probably was doing more timesteps than necessary previously.